astro/src/content/docs/get-started/run-in-the-cloud/disaster-recovery.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Disaster Recovery (DR)
+description: Exploring Disaster Recovery (DR) options in FusionAuth Cloud.
+section: get started
+subcategory: run in the cloud
+---
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import Aside from '/src/components/Aside.astro';
+import { RemoteCode } from '@fusionauth/astro-components';
+import EnterprisePlanBlurb from 'src/content/docs/_shared/_enterprise-plan-blurb.astro';
+import PrimaryServing from 'src/diagrams/docs/get-started/run-in-the-cloud/disaster-recovery/primary-serving.astro';
+import SecondaryServing from 'src/diagrams/docs/get-started/run-in-the-cloud/disaster-recovery/secondary-serving.astro';
+
+<EnterprisePlanBlurb />
+
+## Disaster Recovery (DR)
+
+FusionAuth Cloud supports multi-region disaster recovery (DR). Let's discuss how to enable and use DR in FusionAuth Cloud.
+
+## What Is FusionAuth Cloud Disaster Recovery
+
+DR, also known as cross-region replication (CRR), makes a secure backup copy of your FusionAuth deployment in a datacenter geographically distant from where your primary FusionAuth deployment runs. The database is securely replicated from your primary geographic region to the secondary region. The locations of the primary and secondary regions are configurable.
+
+The primary FusionAuth deployment normally serves all production traffic, and the secondary is kept as a backup.
+
+In the event of a disaster affecting the geographic area of your primary deployment, you can redirect traffic from your primary FusionAuth deployment to your secondary one. When your secondary deployment is fully promoted and DNS changes have propagated, the secondary will serve all production traffic and the primary will serve none.
+
+Using DR minimizes downtime, business impact and user frustration if the data centers running your primary deployment are offline. With FusionAuth Cloud DR, you can redirect login traffic away from a regional failure in minutes.
+
+FusionAuth Cloud DR is an active/passive DR configuration. You are only serving live traffic out of one deployment at a time.
+
+## Requirements
+
+To enable DR:
+
+* your deployment must be highly available (HA)
+* you must purchase an Enterprise plan
+
+In addition, DR is not available without talking to the [technical FusionAuth sales team](/contact). It requires additional configuration and guidance.
+
+## Set Up
+
+After consulting with the technical sales team and purchasing the correct hosting and plan, there are a few other steps.
+
+First, you'll need to determine the secondary region for your DR setup. This can be any region supported by FusionAuth Cloud.
+
+Set up a custom domain name for your FusionAuth instance. This will point to a [durable CNAME provided by FusionAuth](/docs/get-started/run-in-the-cloud/cloud#what-is-a-durable-fusionauth-cname).
```

Comment:
I think we could use stronger wording. The CNAME must point to the durable name and not `example.fusionauth.io`.

Reply:
Hmm. Are you talking about the DR scenario specifically or all custom domain name setups?

Reply:
More generally, if a customer has an existing deployment with custom domains setup without the durable, those DNS records would need to change if they wanted to add a DR deployment.

Reply:
Ah, updated the text to make that clearer.